sdks/go/pkg/beam/core/runtime/graphx/coder.go

```diff
@@ -379,18 +379,14 @@ func (b *CoderUnmarshaller) makeCoder(id string, c *pipepb.Coder) (*coder.Coder,
 			return nil, err
 		}
 		return coder.NewN(elm), nil
-
-		// Special handling for window coders so they can be treated as
-		// a general coder. Generally window coders are not used outside of
-		// specific contexts, but this enables improved testing.
-		// Window types are not permitted to be fulltypes, so
-		// we use assignably equivalent anonymous struct types.
 	case urnIntervalWindow:
```

Comment:
Now that we have a coder* for it, we should ensure that the coder can be round trip serialized. Please add an Unmarshal(Marshal(*coder*)) round trip test (see graphx/coder_test.go)

Reply:
I think that's the only remaining thing for me to block on before merging.

Also, FYI, my foretold change: #24346 
At this point I'll handle the merge conflict instead.

Reply:
Done!